src/models/ValidatedUser.php

```diff
@@ -26,9 +26,9 @@ public static function fromOrgid(string $orgid): Users
         return self::search('orgid', $orgid, true);
     }
 
-    public static function fromExternal(string $email, array $teams, string $firstname, string $lastname): Users
+    public static function fromExternal(string $email, array $teams, string $firstname, string $lastname, ?Usergroup $usergroup = null, bool $automaticValidationEnabled = false, bool $alertAdmin = true, ?string $orgid = null): Users
     {
-        return parent::fromScratch($email, $teams, $firstname, $lastname, null, true);
+        return parent::fromScratch($email, $teams, $firstname, $lastname, $usergroup, $automaticValidationEnabled, $alertAdmin, $orgid);
```

Comment:
We probably don't need all this new parameters for `fromExternal()` just add `?string $orgid = null` and use named arguments in `fromScratch()` (`orgid: $orgid`).

Reply:
you're right, much simpler this way